README.md

```diff
@@ -7,41 +7,43 @@ The data lineages and data quality are tracked automatically between HTTP endpoi
 By default, the `Jaeger backend tracer + Zipkin reporter` is used to keep the spans published to a observability platform, this tracer is combined with the Kensu tracer to collect what is important for Kensu to build lineage and stats.
 
 
-Build and run
-------
-
-Starting Jax-Rs server for sample app (incl Wildflow & MySQL in docker):
-```bash
-( cd ./docker-dir && run-docker.sh )
-./build-dependencies.sh
-./build-and-run-app.sh
-```
-
+One-click run all in Docker
+-----
 
-Some endpoints working out of the box (enough to have just MySQL):
+First run `./one-click-start-all-prebuilt.sh` (to start this app with MySQL & Zipkin inside a docker), and, after waiting a few minutes for it to start up, visit some of these endpoints:
 
 - [http://127.0.0.1:8080/rest/v1/product-line/Motorcycles](http://127.0.0.1:8080/rest/v1/product-line/Motorcycles)
 - [http://127.0.0.1:8080/rest/v1/product-line/Ships](http://127.0.0.1:8080/rest/v1/product-line/Ships)
 - [http://127.0.0.1:8080/rest/v1/order-details/product-line/Ships?maxResults=17](http://127.0.0.1:8080/rest/v1/order-details/product-line/Ships?maxResults=17)
 
-Check on Kensu UI: data catalog, lineages, and projects.
+This will trigger some metadata on data movement and data quality getting collected and sent to Zipkin and/or Kensu servers.
 
+### Check results: Zipkin & Kensu app
 
-Debugging
------
+If you have access to Kensu App, on its UI you may first check - data catalog, lineages, and projects.
 
-ZIPKIN UI (optional, very useful for debugging raw Span data which was sent):
-- run `./docker-dir/start-zipkin.sh`
-- [http://127.0.0.1:9411/zipkin/](http://127.0.0.1:9411/zipkin/)
+While, to understand the internals (debug the raw data sent), visit Zipkin at [http://127.0.0.1:9411/zipkin/?serviceName=sample-service](http://127.0.0.1:9411/zipkin/?serviceName=sample-service) and click on `Run query` - you'll see list of traces/spans that were sent, including info on: http request (url/params), SQL query & stats, and http response (json schema), like this:
 
-Alternatively one could reconfigure it to use JAEGER:
-- Jaeger UI [http://localhost:16686/search](http://localhost:16686/search)
+![zipkin-screenshot](./docs/screenshots/sample-raw-data.png)
 
 
+That's it! All the rest info below is only for advanced techy users wanting to explore the internals.
+
+
+Build it yourself and run
+------
+
+Starting Jax-Rs server for sample app (incl Wildflow & MySQL in docker):
 ```bash
-docker exec -it ksu-wildfly bash
+( cd ./docker-dir && run-docker.sh )
+./build-dependencies.sh
+./build-and-run-app.sh
 ```
 
+Then follow the same steps as in the instructions above
+
+
+
 # Technical details
```

Comment:
maybe to reduce the clutter we may want to move the deep technical info to a separate .md file?

Reply:
Good point. Possible we could use GH Wiki feature?

Reply:
maybe good idea. I'd worry a bit putting docs outside of VCS it might make it hard to manage (already found deps listing to be not yet updated for new code), but it's kinda unstructured anyway.

still could certainly move out these sections:
- Deployment
- Optional: Install Jaeger (there's Docker)
- Optional: Install Wildfly (there's Docker)
- https://github.com/kensuio-oss/jaxrs-sample#maven
- Build https://github.com/kensuio-oss/jaxrs-sample#build
- SSL (on kensu servers)